js/multiFileUpload.js

```diff
@@ -1,351 +1,390 @@
-(function ($)
+(function ($) {
+
+  "use strict";
+
+  Qubit.multiFileUpload = Qubit.multiFileUpload || {};
+
+  var MultiFileUpload = function (element)
```

Comment:
Something that I learnt recently and will make this a bit cleaner. A var is not actually needed, you can name the function directly ... `function MultiFileUpload(element)`. Then, the prototype will extend this function and the function content is assigned directly as the constructor, so it doesn't need to be passed again. Check the latest version of `clipboard.js`.

Reply:
Interesting - I've changed the syntax as you suggested. See latest commits.